docs/GETTING_STARTED.md

```diff
@@ -174,24 +174,27 @@ The server needs access to some of BW's data files in order to generate map imag
 editor, such as [this one](http://www.zezula.net/en/mpq/download.html) and make sure to download
 "listfiles" from that website as well, which you'll need to use in the mpq editor. Use the mpq
 editor to extract BW's data files from `stardat.mpq`, `broodat.mpq`, in that order, having
-`broodat.mpq` overwrite any conflicting files from `stardat.mpq`. The necessary directories in
+`broodat.mpq` overwrite any conflicting files from `stardat.mpq`. 
+You can obtain `stardat.mpq` and `broodat.mpq` directly from Blizzard by downloading the final release of
+[StarEdit](http://download.blizzard.com/pub/starcraft/StarEdit/StarEdit.zip). You can also obtain those files from
+an old Starcraft Broodwar version (such as 1.16) installation folder. Note that you cannot find those files from SC:Remastered.
+The necessary directories in
```

Comment:
Not sure if it affects the final output, but there's an odd line break here.

Reply:
This does not affect the view since you need two blank spaces to represents a line break in Markdown.
I agree it is odd I will fix this.